src/sass/app_2/_common/base/themes.scss

```diff
@@ -1,59 +1,65 @@
-        border_hover_color          : $COLOR_DARK_BLUE_7,
-        border_outline_color        : $COLOR_DARK_BLUE_8,
-        chart_gradient_color        : $COLOR_DARK_BLACK_GRADIENT,
-        container_color             : $COLOR_DARK_BLUE_2,
-        disabled_color              : $COLOR_DARK_BLUE_8,
-        hover_color                 : $COLOR_DARK_BLUE_3,
-        lost_gradient_color         : $COLOR_DARK_RED_GRADIENT,
-        navbar_active_color         : $COLOR_WHITE,
-        navbar_background_color     : $COLOR_DARK_BLUE_5,
-        slider_icon_color           : $COLOR_PURPLE,
-        tab_active_background_color : $COLOR_DARK_BLUE_5,
-        tab_active_color            : $COLOR_WHITE,
-        tab_background_color        : $COLOR_DARK_BLUE_2,
-        tab_hover_color             : $COLOR_DARK_BLUE_4,
-        text_active_color           : $COLOR_WHITE,
-        text_color                  : $COLOR_WHITE,
-        text_primary_color          : $COLOR_DARK_BLUE_7,
-        text_secondary_color        : $COLOR_DARK_BLUE_7,
-        won_gradient_color          : $COLOR_DARK_GREEN_GRADIENT,
+        active_color                     : $COLOR_DARK_BLUE_5,
+        background_button_color          : $COLOR_DARK_BLUE_5,
+        background_color                 : $COLOR_DARK_BLUE_1,
+        background_container_color       : $COLOR_DARK_BLUE_3,
+        background_cover                 : $COLOR_DARK_BLUE_2,
+        background_icon_color            : $COLOR_DARK_BLUE_5,
+        background_tooltip_color         : $COLOR_DARK_BLUE_5,
+        border_hover_color               : $COLOR_DARK_BLUE_7,
+        border_outline_color             : $COLOR_DARK_BLUE_8,
+        chart_gradient_color             : $COLOR_DARK_BLACK_GRADIENT,
+        container_color                  : $COLOR_DARK_BLUE_2,
+        disabled_color                   : $COLOR_DARK_BLUE_8,
+        hover_color                      : $COLOR_DARK_BLUE_3,
+        lost_gradient_color              : $COLOR_DARK_RED_GRADIENT,
+        navbar_active_color              : $COLOR_WHITE,
+        navbar_background_color          : $COLOR_DARK_BLUE_5,
+        slider_icon_color                : $COLOR_PURPLE,
+        tab_active_background_color      : $COLOR_DARK_BLUE_5,
+        tab_active_color                 : $COLOR_WHITE,
+        tab_background_color             : $COLOR_DARK_BLUE_2,
+        tab_hover_color                  : $COLOR_DARK_BLUE_4,
+        text_active_color                : $COLOR_WHITE,
+        text_color                       : $COLOR_WHITE,
+        text_primary_color               : $COLOR_DARK_BLUE_7,
+        text_secondary_color             : $COLOR_DARK_BLUE_7,
+        won_gradient_color               : $COLOR_DARK_GREEN_GRADIENT,
+        purchase_background_color        : $COLOR_DARK_GREY_1,
+        purchase_detail_background_color : $COLOR_DARK_GREY_2,
+        loader_background                : rgba($COLOR_WHITE, 0.16),
```

Comment:
you can have a look at `transparentize` mixin to add transparency

Reply:
What is the advantage of `transparentize` mixin over `rgba`? I think we should be as native as possible.

Reply:
both is fine i guess, however, i think you can make only `$COLOR_WHITE` and `$COLOR_BLACK` in loader background since both having the same `alpha: 0.16`, and then add rgba in your themify. so `background-color: rgba(themed('loader_background), 0.16)'`